docs/sphinx/user_guide/feature/view.rst

```diff
@@ -82,6 +82,44 @@ accesses array entries with unit stride. The loop::
 
 access array entries with stride N :subscript:`n` * N :subscript:`(n-1)` * ... * N :subscript:`(j+1)`.
 
+MultiView
+^^^^^^^^^^^^^^^^
+
+A ``RAJA::MultiView`` object wraps an array-of-pointers,
+or a pointer-to-pointers, rather than a ``RAJA::View`` which wraps a single
+pointer or array. This allows a single ``RAJA::Layout`` to be applied to a
+large set of arrays, and encapsulates a set of arrays more easily.

```

Comment:
"large set" -> "multiple" and "encapsulates a set of arrays" -> "allowing multiple arrays to share indexing arithmetic when their access patterns are the same"

Reply:
This still seems unclear to me. I think simple and direct is better: "applied to multiple arrays." instead.